perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/memberstabs/SendPasswordResetRequestTabItem.java

```diff
@@ -83,12 +83,35 @@ public Widget draw() {
 		layout.setHTML(0, 0, "Select namespace:");
 		layout.getFlexCellFormatter().setStyleName(0, 0, "itemName");
 
-		layout.setHTML(1, 0, "Login:");
+		layout.setHTML(1, 0, "Select language of email:");
 		layout.getFlexCellFormatter().setStyleName(1, 0, "itemName");
 
+		layout.setHTML(2, 0, "Select email attribute:");
+		layout.getFlexCellFormatter().setStyleName(2, 0, "itemName");
+
+		layout.setHTML(3, 0, "Login:");
+		layout.getFlexCellFormatter().setStyleName(3, 0, "itemName");
+
 		final ListBox namespaces = new ListBox();
 		layout.setWidget(0, 1, namespaces);
 
+		final ListBox languages = new ListBox();
+		layout.setWidget(1, 1, languages);
+
+		final ListBox emails = new ListBox();
+		layout.setWidget(2, 1, emails);
+
+		languages.addItem("en");
```

Comment:
It would be nice for user to see full language names, you should be able to get it from native language property too. So the add it to the ListBox like ``box.add(key,value)`` if I remember correctly. So one value is displayed and other is used as backend value.